client/src/components/WeekListItem/WeekListItem.js

```diff
@@ -20,6 +22,9 @@ const StyledDatePicker = styled(DatePicker)`
   :focus {
     outline: none;
   }
+  ::placeholder {
```

Comment:
```suggestion
  :placeholder {
```
Looks like there's one too many colons there :)

Reply:
I'll check on that one...
....as far as I know it has to be written with double ::